src/Components/Alert/Components/Filters/__tests__/Price.jest.tsx

```diff
@@ -0,0 +1,66 @@
+import { fireEvent, render, screen } from "@testing-library/react"
+import {
+  AlertProvider,
+  useAlertContext,
+} from "Components/Alert/Hooks/useAlertContext"
+import {
+  ArtworkFilterContextProvider,
+  useArtworkFilterContext,
+} from "Components/ArtworkFilter/ArtworkFilterContext"
+import { Price } from "Components/Alert/Components/Filters/Price"
+
+describe("PriceFilter", () => {
+  let filterContext
```

Comment:
It might be ok to remove the filter context and related assertions from these tests (see discussion https://github.com/artsy/force/pull/13106#discussion_r1386452733).

Reply:
Addressed, thank you